pkg/cmd/pr/view/view.go

```diff
@@ -180,22 +231,102 @@ func printHumanPrPreview(io *iostreams.IOStreams, pr *api.PullRequest) error {
 	}
 
 	// Body
-	if pr.Body != "" {
-		fmt.Fprintln(out)
-		style := markdown.GetStyle(io.TerminalTheme())
-		md, err := markdown.Render(pr.Body, style, "")
+	fmt.Fprintln(out)
+	if pr.Body == "" {
+		pr.Body = "_No description provided_"
+	}
+	style := markdown.GetStyle(io.TerminalTheme())
+	md, err := markdown.Render(pr.Body, style, "")
+	if err != nil {
+		return err
+	}
+	fmt.Fprint(out, md)
+	fmt.Fprintln(out)
+
+	// Comments
+	if pr.Comments.TotalCount > 0 {
+		comments, err := prCommentList(io, pr.Comments)
 		if err != nil {
 			return err
 		}
-		fmt.Fprintln(out, md)
+		fmt.Fprint(out, comments)
 	}
-	fmt.Fprintln(out)
 
 	// Footer
-	fmt.Fprintf(out, cs.Gray("View this pull request on GitHub: %s\n"), pr.URL)
+	fmt.Fprintf(out, cs.Gray("View this pull request on GitHub: %s"), pr.URL)
+
 	return nil
 }
 
+func prCommentList(io *iostreams.IOStreams, comments api.Comments) (string, error) {
```

Comment:
When rendering PR comments, do you think you could invoke the same method that issue comments are using to render comments to the terminal? That would, of course, involve first extracting issue comments rendering into the `issue/shared` package, or into another package that will effectively be shared between `issue/pr view`.